### PR TITLE
Revise the markdown to GitHub standard and add links for TOC

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
Hi @zaporylie - I am working on mirroring your plugin repository to another place so it can be hosted under here: https://developer.vippsmobilepay.com/docs/vipps-plugins/
But, this one page is looking quite different from the other pages. So I have made a PR to align them a bit better. I hope this will be ok with you. It should be totally trivial changes.

File: README.md [hunk 1]
```diff
@@ -1,65 +1,59 @@
-Commerce Vipps
---------------
+# Commerce Vipps
 
-This module is published on [Drupal.org][Commerce Vipps] and [github.com][Commerce Vipps on GitHub].
+This module is published on [Drupal.org][Commerce Vipps] and [gitub.com][Commerce Vipps on GitHub].
```
Comment (zaporylie) [inline README.md:3]: Typo?

**Review by zaporylie — CHANGES_REQUESTED** [review]
That's ok. Please have a look at the introduced typo, otherwise changes to syntax should be ok for both GitHub and GitLab.

File: README.md (hunk 1 above)
Comment (rebekaburnett) [reply README.md:3]: UhOh! Thank you!

Comment (rebekaburnett) [reply README.md]: Fixed. The lint editor was nagging me to capitalize the H and I did and meant to undo, but lost it instead.

**Comment by rebekaburnett** [comment]
Was I supposed to make the PR to https://git.drupalcode.org/project/commerce_vipps ? I can make a duplicate one there, if you would prefer.

**Pull request merged**